### PR TITLE
Make `SetMaestroVars.yml` set $ci=$true

### PR DESCRIPTION
**Pull request by JohnTortugo**
Make `SetMaestroVars.yml` set $ci=$true so that it's able to use the `Write-PipelineSetVariable` functions.

### CODE REVIEW

**Comment by JohnTortugo** [comment]
Tested this in this build: https://dnceng.visualstudio.com/internal/_build/results?buildId=229618&view=logs&s=f65cc152-4f9d-5670-c98b-f8f01fd59748&j=0c91534b-cdfb-5dcd-5ede-db1be6e1f11c

The variable is now set. The variable content doesn't include the `ChannelID` because there is no default channel for that branch, though.

File: eng/common/templates/post-build/setup-maestro-vars.yml [hunk 1]
```diff
@@ -16,11 +16,23 @@ jobs:
...
+            Write-Host "Asked Write-PipelineSetVariable to create BARBuildId with value '$BarId'"
+
+            $Channels = ""
+            Get-Content "$(Build.StagingDirectory)/ReleaseConfigs/Channels.txt" | ForEach-Object { $Channels += "$_ ," }
+            Write-PipelineSetVariable -Name 'InitialChannels' -Value "$Channels"
+
+            Write-Host "Asked Write-PipelineSetVariable to create InitialChannels with value '$Channels'"
+          }
+          else {
+            Write-Host "This task require AzDO CI environment."
```
Comment (riarenas) [inline eng/common/templates/post-build/setup-maestro-vars.yml]:
```suggestion
            Write-Host "This step can only be run in an Azure DevOps CI environment."
```

File: eng/common/templates/post-build/setup-maestro-vars.yml [hunk 2]
```diff
@@ -16,11 +16,23 @@ jobs:
...
         script: |
-          . "$(Build.SourcesDirectory)/eng/common/tools.ps1"
-             
-          $BarId = Get-Content "$(Build.StagingDirectory)/ReleaseConfigs/BARBuildId.txt" 
-          Write-PipelineSetVariable -Name 'BARBuildId' -Value $BarId
-           
-          $Channels = ""
-          Get-Content "$(Build.StagingDirectory)/ReleaseConfigs/Channels.txt" | ForEach-Object { $Channels += "$_ ," }
-          Write-PipelineSetVariable -Name 'InitialChannels' -Value "$Channels"
+          # This is needed to make Write-PipelineSetVariable work in this context
```
Comment (riarenas) [inline eng/common/templates/post-build/setup-maestro-vars.yml]:
```suggestion
          # This is needed to make Write-PipelineSetVariable works in this context
```

**Comment by riarenas** [comment]
[nit] Include a description for the PR so others who don't have context can weigh in. and When you squash merge, make sure the commit message includes the actual fix, seeing "Patch set maestro variables" in the history doesn't say much.

**Comment by JohnTortugo** [comment]
I'll merge even without CI build completing because this change isn't really tested in CI.

**Pull request merged**